src/features/displayData/index.ts

```diff
@@ -1,16 +1,20 @@
-import type { IAttachments } from 'src/features/attachments';
+import type { AttachmentsSelector } from 'src/features/attachments/AttachmentsStorePlugin';
 import type { IUseLanguage } from 'src/features/language/useLanguage';
-import type { useAllOptionsSelector } from 'src/features/options/useAllOptions';
+import type { NodeOptionsSelector } from 'src/features/options/OptionsStorePlugin';
 import type { FormDataSelector } from 'src/layout';
 import type { CompTypes } from 'src/layout/layout';
 import type { LayoutNode } from 'src/utils/layout/LayoutNode';
+import type { NodeDataSelector } from 'src/utils/layout/NodesContext';
+import type { NodeFormDataSelector } from 'src/utils/layout/useNodeItem';
 
 export interface DisplayDataProps {
-  attachments: IAttachments;
-  optionsSelector: ReturnType<typeof useAllOptionsSelector>;
+  attachmentsSelector: AttachmentsSelector;
+  optionsSelector: NodeOptionsSelector;
   langTools: IUseLanguage;
   currentLanguage: string;
   formDataSelector: FormDataSelector;
+  nodeFormDataSelector: NodeFormDataSelector;
+  nodeDataSelector: NodeDataSelector;
 }
```

Comment:
Veldig bra at "alt" nå er selektorer, liker veldig godt det patternet for app-frontend, siden det er så mange (nødvendige) avhengigheter imellom ting, og da slipper vi masse unødvendige oppdateringer 💪